apps/README.md

```diff
@@ -0,0 +1,11 @@
+# libavif apps dir
+
+This directory contains the source code to the example applications avifenc,
+avifdec, and avifgainmaputil.
+
+## Build Notes
+
+Building avifenc, avifdec, and avifgainmaputil requires:
+* [CMake](https://cmake.org/)
+* [libpng](http://www.libpng.org/pub/png/libpng.html)
+* [libjpg](https://libjpeg.sourceforge.net/)
```

Comment:
nit: libjpeg

Reply:
How about also adding "...or libjpeg-turbo" (often the only available choice in many distros)?